 URL : "https://github.com/huwwp/cryptop/issues/15" TITLE : cannot run 0.1.2 BODY : after upgrading from 0.1.1 to 0.1.2 i get to following error during startup: traceback most recent call last : file /usr/local/bin/cryptop , line 9, in <module> load_entry_point 'cryptop==0.1.2', 'console_scripts', 'cryptop' file /usr/lib/python2.7/dist-packages/pkg_resources/__init__.py , line 542, in load_entry_point return get_distribution dist .load_entry_point group, name file /usr/lib/python2.7/dist-packages/pkg_resources/__init__.py , line 2569, in load_entry_point return ep.load file /usr/lib/python2.7/dist-packages/pkg_resources/__init__.py , line 2229, in load return self.resolve file /usr/lib/python2.7/dist-packages/pkg_resources/__init__.py , line 2235, in resolve module = __import__ self.module_name, fromlist= '__name__' , level=0 file /usr/local/lib/python2.7/dist-packages/cryptop-0.1.2-py2.7.egg/cryptop.py , line 6, in <module> import configparser importerror: no module named configparser